genrl/deep/agents/base.py

```diff
@@ -82,22 +80,24 @@ def __init__(
         network_type: str,
         env: Any,
         batch_size: int = 256,
-        layers: Tuple = (64, 64),
         gamma: float = 0.99,
+        layers: Tuple = (64, 64),
         lr_policy: float = 0.01,
         lr_value: float = 0.0005,
-        epochs: int = 100,
         rollout_size: int = 2048,
         **kwargs
     ):
-        super(OnPolicyAgent, self).__init__(network_type, env, epochs, **kwargs)
-        self.batch_size = batch_size
-        self.gamma = gamma
+        super(OnPolicyAgent, self).__init__(
+            network_type,
+            env,
```

Comment:
These two can be replaced with `*args`?

Reply:
I think that would get a little confusing and vague to users because the docs for that would be gone too. Are you suggesting we replace the init with `__init__(*args, **kwargs) `?

Reply:
Docs are independent of this change? Will depend only on the docstring?

Reply:
Yeah essentially all arguments from super's init can be managed with args, kwargs. So you can avoid network_type and env and anything else present in there

Reply:
Oh okay, I misunderstood. I'll do it soon.